lib/gds_api/publishing_api.rb

```diff
@@ -579,9 +579,11 @@ def get_schema(schema_name)
   #
   # @param query [String]
   #
-  # @return [GdsApi::Response] A response with the result of the GraphQL query.
+  # @return [GdsApi::Response] A response with the result of the GraphQL query whose keys are changed to be underscored.
   def graphql_query(query)
-    post_json("#{endpoint}/graphql", query:)
+    response = post_json("#{endpoint}/graphql", query:)
```

Comment:
Repeating what I said in directly to Bruce, for the rest of the team: I reckon we could do something with the `JsonClient`'s `create_response` block (https://github.com/alphagov/gds-api-adapters/blob/0ddad8bcd864c0f39a521988c0ff4d5f29cd6301/lib/gds_api/json_client.rb#L93-L94). It'd allow us to be as Publishing API- / GraphQL-specific as we wanted without forcing those details onto the rest of GDS API Adapters.